packages/analytics/dashboard-renderer/sandbox/pages/RendererDemo.vue

```diff
@@ -153,7 +153,6 @@ const dashboardConfig: DashboardConfig = {
     {
       definition: {
         chart: {
-          chartTitle: 'Timeseries line chart of mock data',
```

Comment:
Minor, but please remove this change; we might still trigger a new renderer version because it depends on `analytics-chart`, but it's nice to avoid code changes when possible.